dm/worker/source.yaml

```diff
@@ -7,13 +7,14 @@ server-id: 101
 source-id: mysql-replica-01
 
 #flavor: mysql/mariadb
-flavor: mysql
 
 #directory that used to store relay log
 relay-dir: ./relay_log
 
 #enable gtid in relay log unit
-enable-gtid: false
+enable-gtid: true
+
+relay-binlog-gtid: "e68f6068-53ec-11eb-9c5f-0242ac110003:1-50"
```

Comment:
there's a case that user didn't specify flavor or gtid, expecting starting relay at position of `SHOW MASTER STATUS`. Will `SHOW MASTER STATUS` returned empty `Executed_Gtid_Set`? If so, error message of "empty flavor with empty gtid is invalid" is misleading because he could only left them empty to use position of `SHOW MASTER STATUS`

Reply:
The `flavor` should not be empty since we have adjusted it or specify it, otherwise it is a bug. So `empty flavor with empty gtid is invalid` should not happen.